### ISSUE TITLE
Feature: add account by public key

### ISSUE THREAD

**Issue by ddombrowsky** [satoshipay/solar#1235]
Right now I cannot to add an account by public key only.  This is useful in two main cases that I can think of:

* view only accounts
* accounts where you have signing power, but no secret key

The 2nd case is most useful for me. 

As a work-around, is there a way to add this account manually to a config file somewhere?

**Comment by andywer** [comment]
Hey David!

I agree, but the first case doesn't have much priority, I'm afraid. For the multi-sig use case I have got good news for you!

Just today we merged the new multi-signature implementation that we will release in the next few days.
You will have to delete your co-signing accounts and re-import your their keys as dedicated co-signing key pairs, but you will then

* not need to own any XLM to be co-signer and
* you will "feel" no difference to having the actual multi-sig account (the account's master key pair) in your wallet.

The co-signer sees the balances, the transactions, can initiate payments, … just as if they created the multi-sig account all along.
